packages/esm-patient-registration-app/src/patient-registration/input/basic-input/select/select-input.test.tsx

```diff
@@ -29,4 +29,16 @@ describe.skip('select input', () => {
 
     await waitFor(() => expect(input.value).toEqual(expected));
   });
+  it('should show optional label if the input is not required', () => {
```

Comment:
```suggestion

  it('should show optional label if the input is not required', () => {
```

can you add a space before the test?

Reply:
ya sure. updated it @anjula-sack